Gemfile.lock

```diff
@@ -211,3 +208,6 @@ DEPENDENCIES
   sqlite3
   standard
   toxiproxy
+
```

Comment:
Remove this and Bundler version doesn’t matter

Reply:
Done.